docs/versions.json

```diff
@@ -1,12 +1,15 @@
-[
```

Comment:
why 'json' if in both cases we use it in `ts` files?

Reply:
`VersionService` from the docs app also requests this file over http

Reply:
Keeping it as json, as discussed irl, since running docs app requesting this file over the http